diff --git a/compat b/compat
index ec63514..45a4fb7 100644
--- a/compat
+++ b/compat
@@ -1 +1 @@
-9
+8
diff --git a/control b/control
index f5cc651..ab4a031 100644
--- a/control
+++ b/control
@@ -1,14 +1,14 @@
 Source: suricata
 Section: net
 Priority: optional
-Maintainer: Pierre Chifflier <pollux@debian.org>
-Build-Depends: debhelper (>= 9),
+Maintainer: Lou Ruppert <himself@louruppert.com>
+Build-Depends: debhelper (>= 8),
     autotools-dev,
     libhtp-dev (>= 0.2.3),
     libnet1-dev | libnet-dev,
     libnspr4-dev,
     libnss3-dev,
-    libpcap-dev,
+    libpcap-dev | libpcap1.1-dev,
     libpcre3-dev,
     libyaml-dev,
     libnetfilter-queue-dev,
@@ -16,8 +16,6 @@ Build-Depends: debhelper (>= 9),
     zlib1g-dev | libz-dev,
     libcap-ng-dev,
     libmagic-dev,
-    libjansson-dev,
-    libluajit-5.1-dev [i386 amd64 powerpc mips armel armhf],
     python
 X-Python-Version: >= 2.6
 Standards-Version: 3.9.4
diff --git a/rules b/rules
index e4ee588..35744cd 100755
--- a/rules
+++ b/rules
@@ -17,14 +17,14 @@ DEB_DESTDIR = $(CURDIR)/debian/suricata
 ARCH ?= $(shell dpkg --print-architecture)
 DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
-LUAJIT_ARCHS = i386 amd64 powerpc mips armel armhf
+LUAJIT_ARCHS = ''
 
 ifneq (,$(findstring $(ARCH),$(LUAJIT_ARCHS)))
 	ENABLE_LUAJIT="--enable-luajit"
 endif
 
-CONFIGURE_ARGS = --enable-af-packet --enable-nfqueue --enable-prelude \
-	--enable-non-bundled-htp --enable-gccprotect --disable-gccmarch-native \
+CONFIGURE_ARGS = --enable-af-packet --enable-nfqueue --enable-pfring --enable-prelude \
+	--enable-gccprotect --disable-gccmarch-native \
 	--with-libnss-includes=/usr/include/nss --with-libnss-libraries=/usr/lib/$(DEB_HOST_MULTIARCH) \
 	--with-libnspr-includes=/usr/include/nspr --with-libnspr-libraries=/usr/lib/$(DEB_HOST_MULTIARCH) \
 	$(ENABLE_LUAJIT)
diff --git a/suricata.default b/suricata.default
index 0eb31b0..059862b 100644
--- a/suricata.default
+++ b/suricata.default
@@ -1,26 +1,26 @@
 # Default config for Suricata
 
 # set to yes to start the server in the init.d script
-RUN=no
+RUN=yes
 
 # Configuration file to load
-SURCONF=/etc/suricata/suricata-debian.yaml
+SURCONF=/etc/suricata/suricata.yaml
 
-# Listen mode: pcap, nfqueue or af-packet
+# Listen mode: pcap or nfqueue
 # depending on this value, only one of the two following options
-# will be used (af-packet uses neither).
+# will be used
 # Please note that IPS mode is only available when using nfqueue
-LISTENMODE=nfqueue
+LISTENMODE=pfring
 
 # Interface to listen on (for pcap mode)
-IFACE=eth0
+IFACE=eth1
 
 # Queue number to listen on (for nfqueue mode)
-NFQUEUE=0
+#NFQUEUE=0
 
 # Load Google TCMALLOC if libtcmalloc-minimal0 is installed
 # This _might_ give you very very small performance gain....
 TCMALLOC="YES"
-
 # Pid file
 PIDFILE=/var/run/suricata.pid
+
diff --git a/suricata.init b/suricata.init
index 42f3c39..f3b6057 100644
--- a/suricata.init
+++ b/suricata.init
@@ -70,6 +70,10 @@ case "$LISTENMODE" in
     IDMODE="IDS (pcap)"
     LISTEN_OPTIONS=" -i $IFACE"
     ;;
+  pfring)
+    IDMODE="IDS (pfring)"
+    LISTEN_OPTIONS=" --pfring"
+    ;;
   af-packet)
     IDMODE="IDS (af-packet)"
     LISTEN_OPTIONS=" --af-packet"
